servers/src/test/java/tachyon/worker/block/meta/StorageDirTest.java

```diff
@@ -49,11 +57,101 @@
   public void before() {
     TachyonConf tachyonConf = new TachyonConf();
     mTier = new StorageTier(tachyonConf, 0 /* level */, 1 /* alias */);
-    mDir = new StorageDir(mTier, TEST_DIR_INDEX, TEST_DIR_CAPACITY, TEST_DIR_PATH);
+    mDir = StorageDir.newStorageDir(mTier, TEST_DIR_INDEX, TEST_DIR_CAPACITY, TEST_DIR_PATH);
     mBlockMeta = new BlockMeta(TEST_BLOCK_ID, TEST_BLOCK_SIZE, mDir);
     mTempBlockMeta = new TempBlockMeta(TEST_USER_ID, TEST_TEMP_BLOCK_ID, TEST_BLOCK_SIZE, mDir);
   }
 
+  private StorageDir newStorageDir(File testDir) {
+    return StorageDir.newStorageDir(mTier, TEST_DIR_INDEX, TEST_DIR_CAPACITY,
+        testDir.getAbsolutePath());
+  }
+
+  private File newBlockFile(File dir, String name, int lenBytes) throws IOException {
```

Comment:
The returned value of `newBlockFile` is not really used in the following source code. Then in this case, why bother returning a `File` instance?